devel/libcryptopp/Portfile

```diff
@@ -40,9 +39,15 @@ proc _get_cryptopp_init_priority_flag {compiler} {
     return ""
 }
 
+# https://trac.macports.org/ticket/69597
+if {${configure.cxx_stdlib} eq "libc++"} {
+    configure.cxxflags-append \
+                    -stdlib=libc++
+}
```

Comment:
Why is this part needed? `${configure.cxxflags}` already contains any needed -stdlib option.

Reply:
I am not sure how the base behaves here. If you say so, let’s remove this chunk.

Reply:
If you're ever unsure of what a variable contains, you can temporarily add something like `ui_msg ${configure.cxxflags}` to the portfile to check.

Reply:
Oh, thank you, didn’t know about this.